From 9af68008d3ddc0f8deba2ef1f7fada8505779716 Mon Sep 17 00:00:00 2001
From: Adeel Arshad <adeel_arshad@mentor.com>
Date: Fri, 7 Oct 2016 12:15:26 +0500
Subject: [PATCH] cross compile compatibility

remove the fixed path that is used to set the location to
look for python modules, we will set it by PYTHONPATH.

Signed-off-by: Adeel Arshad <adeel_arshad@mentor.com>
---
 Common/Src/SCons/CXL_init.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Common/Src/SCons/CXL_init.py b/Common/Src/SCons/CXL_init.py
index d0a10a1..bf36509 100755
--- a/Common/Src/SCons/CXL_init.py
+++ b/Common/Src/SCons/CXL_init.py
@@ -10,7 +10,6 @@ import platform
 
 def initCXLVars (CXL_vars) :
     try:
-        sys.path.append("/usr/lib/scons/SCons/Variables/")
         enum_mod = __import__("EnumVariable")
     except ImportError:
         print "Error: Could not import EnumVariable"
-- 
1.9.1

